api/v1/nodeptpdevice_types.go

```diff
@@ -21,47 +21,73 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+//+kubebuilder:printcolumn:name="HwConfig",type="string",JSONPath=".status.hwconfig.status"
```

Comment:
HwConfig is an array, not string

Reply:
gotcha, will remove the proposed new column